Sources/Class-CurlFetchWeb.php

```diff
@@ -21,21 +21,20 @@
  *
  * Load class
  * Initiate as
- *  - $fetch_data = new cURL_fetch_web_data();
- *	- optionally pass an array of cURL options and redirect count
- *	- cURL_fetch_web_data(cURL options array, Max redirects);
- *  - $fetch_data = new cURL_fetch_web_data(array(CURLOPT_SSL_VERIFYPEER => 1), 5);
+ *   <code>$fetch_data = new cURL_fetch_web_data();</code>
+ *   optionally pass an array of cURL options and redirect count
+ *   <code>$fetch_data = new cURL_fetch_web_data(array(CURLOPT_SSL_VERIFYPEER => 1), 5);</code>
  *
  * Make the call
- *  - $fetch_data('https://www.simplemachines.org'); // fetch a page
- *  - $fetch_data('https://www.simplemachines.org', array('user' => 'name', 'password' => 'password')); // post to a page
- *  - $fetch_data('https://www.simplemachines.org', parameter1&parameter2&parameter3); // post to a page
+ *   <code>$fetch_data->get_url_data('https://www.simplemachines.org'); // fetch a page</code>
```

Comment:
It would be better to put the trailing comments on these lines outside the <code> tags.